docs/ml-guide.md

```diff
@@ -801,6 +801,173 @@ jsc.stop();
 
 </div>
 
+## Example: Model Selection via Train Validation Split
+In addition to  `CrossValidator` Spark also offers
```

Comment:
I would cut L805-809 since L811-819 says essentially the same thing but is better IMO